tests/testCameraGeom.py

```diff
@@ -218,6 +218,8 @@ def testTransformDet(self):
 
     def testFindDetectors(self):
         for cw in self.cameraList:
+            detXPoints = []
+            detYPoints = []
```

Comment:
Use detPointList and don't bother to split into x and y. If you improve findDetectorsArray as suggested earlier then you won't need the split.